package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "witnet-solidity",
-  "version": "2.0.0-beta.0",
+  "version": "2.0.0-beta",
```

Could we keep track of the beta versions?
```suggestion
  "version": "2.0.0-beta.7",
```